api/python/briefcase/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[project]
+name = "briefcasex-slint"
```

Comment:
Is the `x` on purpose or is it a typo?
Is it how briefcase plugin are usually named?
If it was me, i'd name it `slint-briefcase`, bu sonsitency with all our slint-* rust crates.